corehq/apps/es/tests/test_esquery.py

```diff
@@ -314,6 +314,14 @@ def test_scroll_ids_without_query_size_uses_default_scroll_size(self):
         with patch("corehq.apps.es.es_query.scroll_query", scroll_query_testfunc):
             list(query.scroll_ids())
 
+    def test_scroll_with_aggregations_raises(self):
+        query = HQESQuery('forms').terms_aggregation('domain.exact', 'domain')
+        with (
+            patch("corehq.apps.es.es_query.scroll_query"),
```

Comment:
is this patch necessary?

Reply:
Yes, albeit for unclear reasons. It prevents the actual scroll from hitting ES, which raises an error due to the index not being setup. I think a try/except would be much clearer, I'll fix that.

Reply:
Surely it won't hit the actual scroll query since the exception gets raised first?

Reply:
Actually, you're right @snopoke, it's not needed at all. I originally added it to make my pre-bugfix-test fail _for the expected reason_ (rather than fail due to an unexpected error), but in hindsight, that's not an important thing to guard for. Thanks for pointing this out.

Reply:
Fixed in 3eda52d.

Reply:
> Surely it won't hit the actual scroll query since the exception gets raised first?

Just saw this now. You are correct.  I added this test _before_ I made the fix, and in my head I wanted it to fail for the expected reason (that is: scroll query succeeds, `InvalidQueryError` not raised), which resulted in an overkill, crufty test.